server.js

```diff
@@ -1,13 +1,25 @@
-app.post('/slambook', async (req, res) => {
+app.put('/slambook/:id', async (req, res) => {
+  const entryId = req.params.id;
+
   try {
-    
-    const newEntry = new SlamBook(req.body);
+   
+    const updatedEntry = await SlamBook.findByIdAndUpdate(
+      entryId,
+      req.body,
+      { new: true }

```

Comment:
Consider returning 'res' after sending the response to avoid unnecessary code execution.